procedures/prs.md

```diff
@@ -48,3 +48,36 @@ See updated changelog file and/or add any other summarized helpful information f
 That's it. Once your PR is submitted make sure to keep an eye on your PR notifications for any follow ups, once it's approved, a core team member will merge it.
 
 Thanks for contributing to Kytos-ng.
+
+## Creating a Backport
+
+Backports are necessary to keep old versions updated. When it is agreed to apply changes to released deployments, it is done for two main reasons:
+- Bugs found: If a bug is found in an older deployment, it is necessary to address issues like this so as not to interrupt production scenarios.
+- New required features: Some features can be desired to be in older versions so it is easier to interact with Kytos.
+
+### How to create a backport and new tagged version
+
+- Find the latest version, e.g. `20XX.2.0`, then checkout that tag `20XX.2.0`.
+- Create and push a new base branch incrementing the patch version `base/20XX.2.1` (only a core dev or maintainer can push a branch upstream, if such titles are needed do not hesitate to request them).
+- Create another branch `release/20XX.2.1` from `base/20XX.2.1`.
+- Create a pull request (PR) and target the branch `base/20XX.2.1`. New changes will be first in `release/20XX.2.1` and will get merged to `base/20XX.2.1`.
+- After the PR is closed, make sure to delete `release/20XX.2.1` branch.
+- Draft a new release with the name as `20XX.2.1`, choose the tag `20XX.2.1` and target the branch `base/20XX.2.1` from this example.
+
+<div align="center">
+  <a href="assets/tag_example.png"><img src="assets/tag_example.png"></a>
+</div>
+
+- Double check if the new lease is targeting the correct branch. To do so, go to the Github release page, open the newly created release, and open the commit. Verify that the last PR was made on the PR which should target `base/20XX.2.1`. For example, from the version `2023.2.8` this is the [commit](https://github.com/kytos-ng/mef_eline/commit/9fad3dd2d5c776b849d3d87cd9bc261174d92add) that can be accessed from the release page (picture from released version).
+
+<div align="center">
+  <a href="assets/version_example.png"><img src="assets/version_example.png"></a>
+</div>
+
+- Send a notification about the new release on the Slack channel.
+
+### Final observations.
+
+PR created for a previous version does not run automatic tests so make sure to run unit tests and end-to-end tests to assure that the changes are safe to be merged.
```

Comment:
Alright, let's also make unit tests mandatory for backports but only running locally on tox, for e2e tests check out the discussion here https://github.com/kytos-ng/kytos-ng.github.io/issues/48#issuecomment-2546402317, e2e tests definitely needs to be run, but before we ask for it too, our https://github.com/kytos-ng/kytos-end-to-end-tests test suites need to also have compatibility versions too such as 2023.2, 2024.1 and so on, and needs to be bumped with the final release too, just so older tests still pass, since newer ones might or might not be fully compatible. So, we need to also map that effort and make sure that part is ready for 2023.2 and 2024.1 if we'll ask for it, can you take a lead on this and map an issue?

Reply:
Issue created, https://github.com/kytos-ng/kytos-end-to-end-tests/issues/340